pages/ecosystem/index.vue

```diff
@@ -130,14 +130,29 @@ import { GeneralLink } from '~/constants/appLinks'
   methods: {
     getTestRows (member: any): void {
       if (member.testsResults) {
-        return member.testsResults.map(
-          ({ terraVersion, testType, passed, timestamp }: any) => {
-            timestamp = new Date(timestamp * 1000).toLocaleString('en-UK', {
+        return member.testsResults.map((res: any) => {
+          // make tiemstamp human readable
+          const timestamp = new Date(res.timestamp * 1000).toLocaleString(
+            'en-UK',
+            {
               timeZone: 'UTC'
-            })
-            return { terraVersion, testType, passed, timestamp }
+            }
+          )
+          // Convert package name to title case
+          const packageName = res.package
+            .replace(/-/g, ' ')
```

Comment:
You can use [replaceAll](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replaceAll) instead of using that Regex